drawSvg/raster.py

```diff
@@ -1,11 +1,18 @@
 
 import io
+import warnings
+from .missing import MissingModule
+
 
 try:
     import cairosvg
-except (ImportError, OSError):
-    import warnings
-    from .missing import MissingModule
+except OSError as e:
+    msg = e.args[0]
+    if msg.startswith("no library called \"cairo\" was found"):
```

Comment:
```suggestion
    if msg.startswith('no library called "cairo" was found'):
```

The error message reported in #4 was "OSError: dlopen() failed to load a library: cairo / cairo-2 / cairo-gobject-2".  Are there different error messages?

Reply:
I reread the error message in #4 and it looks like that that person was using an Anaconda environment to load drawSvg. When I tested my error message, I used iPython and printed out the error message:

```
no library called "cairo" was found
no library called "libcairo-2" was found
cannot load library 'libcairo.so': dlopen(libcairo.so, 2): image not found
cannot load library 'libcairo.2.dylib': dlopen(libcairo.2.dylib, 2): image not found
cannot load library 'libcairo-2.dll': dlopen(libcairo-2.dll, 2): image not found
```

I initially was hesitant to use `startswith` to catch the cairo error message because it is too restrictive in it search. But I decided to use `startswith` instead of `find` because find seemed too expensive an operation to determine if an error message is related to Cairo. 

But given your question and given that drawSvg is used on multiple platforms and that there appears to be no guarantee that a Cairo-related error message will start with "no library called cairo was found", I suggest using `find` to capture a cairo-related message:

* error message from iPython
```
In [10]: msg
Out[10]: 'no library called "cairo" was found no library called "libcairo-2" was found cannot load library \'libcairo.so\': dlopen(libcairo.so, 2): image not found cannot load library \'libcairo.2.dylib\': dlopen(libcairo.2.dylib, 2): image not found cannot load library \'libcairo-2.dll\': dlopen(libcairo-2.dll, 2): image not found'

In [11]: msg.find('cairo')
Out[11]: 19
```

* error message from #4 
```
In [15]: msg = "OSError: dlopen() failed to load a library: cairo / cairo-2 / cairo-gobject-2"

In [16]: msg.find('cairo')
Out[16]: 44
```

What are your thoughts?

Reply:
Now that I've thought about it, I think checking the content of the message is a bad idea.  It would be more robust to print the same warning for any OSError but also include the original error message.  Something like
```
msg = ('Failed to import CairoSVG.  '
       'drawSvg will be unable to output PNG or other raster image formats.  '
       'See https://github.com/cduck/drawSvg#prerequisites for more details.\n'
       'Original OSError: {}'.format(e))
```

Reply:
Agree. Including the original error message is a good idea and will help prevent masking the root cause of an OSError. Will change.